grpc.yaml

```diff
@@ -5,6 +5,10 @@ package:
   description: The C based gRPC
   copyright:
     - license: Apache-2.0 AND BSD-3-Clause AND MIT
+  dependencies:
+    runtime:
+      - libprotobuf
+      - libprotoc
   resources:
```

Comment:
this is redundant, as protobuf and grpc abi transition is resolve via no change rebuild.

Please drop any grpc changes.

Reply:
revert all grpc changes